### DISCUSSION TITLE
Census variables

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/earth-analytics-02-python-2024#3] [Q&A01 Urban Greenspace | Cloud Data]
In your assignment notebook, the instructions originally say to download median income for each tract. However, later on the instructions refer to three other variables: population, per capita income, and median year built. You should feel free to work with whichever variables you like, but those three will help illustrate some points about the linear OLS statistical model.

**Comment by edchanonline** [comment]
Does that mean I could use the model to predict per capita income and median income?? I'm looking at scatter plots and scatter matrixes, and I don't really notice any significant relationships between NDVI and these variables. Just want to make sure I don't need to use any NDVI variable. Thanks!

**Reply by edchanonline** [reply]
To illustrate what I see:

![Screenshot 2024-01-28 at 8 32 36 PM](https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/4795392/c4ed1a03-d033-437e-8372-68fbaa8bd367)
![Screenshot 2024-01-28 at 8 32 46 PM](https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/4795392/76e87397-32d8-4dd4-8f3d-29a06119aa42)

Do let me know if I have missed something obvious! Statistics is not my forte!!

**Reply by edchanonline** [reply]
Ignore me! I am re-reading the following again:
> One way to find if there is a statistically significant relationship between the socioeconomic parameters from the U.S. Census and greenspace as measured by the fraction of pixels with an NDVI greater than .12 is to run a linear ordinary least squares (OLS) regression and measure how well it is able to predict greenspace given your chosen socioeconomic variables.
